src/Home.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ModulePlan from './ModulePlan';
+import moduleplan from './moduleplan.json';
+
+class Home extends React.Component {
+  render() {
+    var modules = moduleplan.degree_course.modules;
+    var maxSemester = 6;
```

Comment:
Ah sorry, I forgot two words in my review message. I was referring to the variable `maxSemester`. That variable is not used in `render`.